packages/php-table-backend-utils/src/Schema/Snowflake/SnowflakeSchemaReflection.php

```diff
@@ -206,4 +232,36 @@ public function getDefinitions(): array
         }
         return $definitions;
     }
+
+    private function fallbackColumnType(string $tableName, string $columnName): SnowflakeColumn
```

Comment:
zaslůžilo by si to vysvětlit proč takový fallback máme.

Reply:
Ono už to tam je, ale jen pro BINARY: https://github.com/keboola/storage-backend/pull/188/files#diff-26e43ac471cdecbc6e355d9922b89e2b8ebaee776a606d3d59c9843560921b1aR115

A docela daleko. Dopíšu to i někam k té funkci. Nebo možná k tomu poli typů, co potřebují fallback.

Reply:
jo komentář v tom SQL mě ani nenapadlo hledat. Chce to tam enom vysvětlit

Reply:
jasně, už to jdu psát

Reply:
je to tu